openfl-tutorials/interactive_api/PyTorch_MedMNIST_3D/requirements.txt

```diff
@@ -0,0 +1,4 @@
+medmnist
+pip install ACSConv
```

Comment:
should be only ACSConv instead of pip install ACSConv. Also, I don't think this package is needed at envoy side but please check once.